### DISCUSSION TITLE
Typeform Replacement for Surveys

### DISCUSSION THREAD

**Discussion by andrashee** [socialincome-san/public#309] [Archived Discussions]
On the long run, we would like to replace Typeform as survey tool. According to discussions (I haven't used it by myself) it has the following downsides:
- missing multi-language support
- expensive
- @ssandino what else is bothering you?

I found two frameworks which seem like a good fit for us:
- https://surveyjs.io: https://surveyjs.io/form-library/documentation/get-started-react. React support, I think multilanguage can be implemented in the same way as we do it for the email / website using i18n-next. Haven't tried it out, but it feels quite solid. Here is an online editor to create the configuration json: https://surveyjs.io/create-free-survey
- https://github.com/quillforms/quillforms: wanted to try it out but was confronted with some issues: https://github.com/quillforms/quillforms/issues/102. Reaction on the issue is slow, so I think I would first try the one mentioned above.

Thoughts?

**Comment by ssandino** [comment]
Some additional drawbacks: 

- the data from the surveys is meanwhile saved into many csv files (as every little change in the questionnaire, starts a new file). It is still possible to bring together all data, but it's always a manual job – which would make it impossible to ever publish the results in real time (impact page).
- it's "hacky" how the mobile app can embed the individual survey and know how it's filled out.
- some of the answers could be also used on our transparency page (like how many people are currently working and how many without a job). If we save the data with us it would be seamless to use it in transparency stats.
- One goal is to publish the data as open data (without revealing names etc.)

**Comment by ssandino** [comment]
Previous discussion (archived): #94 (just for reference)
